client/app/stories/Toast.stories.jsx

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React from 'react';
+import { Toast } from 'components/Toast';
+
+export default {
+  title: 'Components/Toast',
+  component: Toast,
+};
+
+const Template = (args) => <Toast {...args} />;
+
+export const noticeToast = Template.bind({});
+
```

Comment:
I don't have much knowledge about stories, but according to me we have to add another case where we are performing a logout scenario: 'Logout successfully'

Reply:
@MuhammadAakash We are using Storybook JS in this project which allows us to build UI components in isolation and capture UI variations as “stories”. So this toast component has actually only TWO states; `alert` & `notice`. The logout scenario is covered under the `notice` state with just a different message. So I think there is no need to document this separately.
You can read more about the Storybook JS [here](https://storybook.js.org/docs/react/why-storybook) and share your thoughts.

Reply:
@zeeshansarwar38 I don't know much about Storybook js that's why I have added that comment.
So, now this is clear.